src/PrestaShopBundle/ApiPlatform/Resources/CustomerGroup.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShopBundle\ApiPlatform\Resources;
+
+use ApiPlatform\Metadata\ApiProperty;
+use ApiPlatform\Metadata\ApiResource;
+use ApiPlatform\Metadata\Post;
+use PrestaShop\Decimal\DecimalNumber;
+use PrestaShop\PrestaShop\Core\Domain\Customer\Group\Command\AddCustomerGroupCommand;
+use PrestaShopBundle\ApiPlatform\Processor\CommandProcessor;
+
+#[ApiResource(
+    operations: [
+        new Post(
+            uriTemplate: '/customers/group',
+            provider: CommandProcessor::class,
+            extraProperties: [
+                'command' => AddCustomerGroupCommand::class,
+            ],
+        ),
+    ]
+)]
+class CustomerGroup
+{
+    /**
+     * @param string[] $localizedNames
+     * @param int[] $shopIds
+     */
+    public function __construct(
+        #[ApiProperty(
+            example: 'array<string>',
+        )]
+        public array $localizedNames,
+        #[ApiProperty(
+            openapiContext: [
+                'example' => [
+                    'exponent' => 'int',
+                    'number' => 'string',
+                ],
+            ],
+        )]
```

Comment:
This is not working. Apparently there is currently no simple way to overide the example of a nested object.
It should be possible to tinker with the serialization groups for this resource in order to impact the example of the documentation.

Reply:
currently the documentation shows 
```
  "reductionPercent": {
    "exponent": 0
  },
```
It should show 
```
  "reductionPercent": {
    "number": "string",
    "exponent": "int"
  },
  ```